src/transformers/models/fsmt/modeling_fsmt.py

```diff
@@ -675,6 +694,7 @@ def forward(
         decoder_padding_mask: torch.Tensor,
         decoder_causal_mask: torch.Tensor,
         head_mask: Optional[torch.Tensor] = None,
+        inputs_embeds: Optional[torch.Tensor] = None,
```

Comment:
Doctring is missing here

Reply:
Fixed in [d3800fd](https://github.com/huggingface/transformers/pull/20825/commits/d3800fdee2cbe30440028ca0932fe1529f4b895b)